installer/src/panels/com/izforge/izpack/installer/AutomatedInstaller.java

```diff
@@ -108,7 +125,49 @@ public AutomatedInstaller(String inputFilename) throws Exception
         // load dynamic variables
         loadDynamicVariables();
     }
-
+    
+    /**
+     * updateRuntimeLocations
+     * Establish the runtime_locations.properties file based on the JBoss Application Server location
+     * (if that panel is active) or minimally just the 'runtimes' directory.  This file is read at startup
+     * and affects the runtime UI preferences.
+     */
+    private void updateRuntimeLocations()
+    {
+    	String installPath = idata.getVariable("INSTALL_PATH");
+		File folder = new File(installPath, P2DirectorStarterListener.DEVSTUDIO_LOCATION);
+
+		Properties servers = (Properties) idata.getAttribute("AS_SERVERS");
+		if (servers == null || servers.isEmpty()) {
+			servers = new Properties();
+			File runtimesFolder = new File (installPath, "runtimes");
+			RuntimePath runtimesPath = new RuntimePath(runtimesFolder.getAbsolutePath(), true);
+			servers.put("server1", runtimesPath.toString());
+		}
+		if (!folder.exists())
+			folder.mkdirs();
+
+		File file = new File(folder, "runtime_locations.properties");
+		FileOutputStream stream = null;
+		try {
```

Comment:
@jeffmaury - removed unneeded file.createNewFile() statement - left the rest as is - couldn't use Java try-with-resources without bumping the source to 7 
ref:
`/home/pleacu/git-clone/jbdevstudio-product/installer/src/panels/com/izforge/izpack/installer/AutomatedInstaller.java:151: error: try-with-resources is not supported in -source 1.5
		try (FileOutputStream stream = new FileOutputStream(file)) {
		    ^
  (use -source 7 or higher to enable try-with-resources)`